views/admin/new_preset_event.erb

```diff
@@ -36,6 +36,12 @@
                     <label for="max-signups"><span class="dcf-sr-only">Max Signups</span></label>
                     <input value="<%= preset_event.max_signups %>" oninput="this.value = this.value.replace(/[^0-9]/g, '').replace(/(\..*?)\..*/g, '$1').replace(/^0[^.]/, '0');" type="number" id="max-signups" name="max_signups" class="dcf-d-inline dcf-w-10" aria-label="Number of signups to limit."/>
                 </div>
+
+                <div class="dcf-form-group">
+                    <div class="dcf-input-checkbox">
```

Comment:
I don't think we need this checkbox on the preset events page since we didn't implement any functionality for setting the privacy for preset events.